### PR TITLE
Reduce flake8 max-line-length

### PR DESCRIPTION
**Pull request by Insharamin12**
# Description

Reduced flake8 maximum length from 155 to 154 and changed some line exceeding the maximum line length. 

# Changed Behaviour

This won't change any functionality, it just improves the readability.

# Fixes

Fixes #3105

## Type of change

Choose which options apply, and delete the ones which do not apply.

- Bug fix
- Update to human readable text: Documentation/error messages/comments
- Code maintenance/cleanup

### CODE REVIEW

**Comment by benclifford** [comment]
thanks for this PR - your first commit looks good. then you merged in master, where someone else had already reduced the line length to 154 (in commit #3130) in a slightly different way - so now there isn't much left of your PR! but thats ok.

so you can make the line length even smaller, to end up with a merged PR for issue #3105 - or you can move on to looking at the issues tagged `outreachy`: https://github.com/Parsl/parsl/issues?q=is%3Aissue+is%3Aopen+label%3Aoutreachy to look for stuff you'd like to learn more about/work on.

**Comment by Insharamin12** [comment]
Thank you for the feedback, @benclifford 

Sure, I'll make the further line change in the issue #3105 and then proceed to other issues tagged  as `outreachy.`

**Comment by Insharamin12** [comment]
Hey @benclifford 

I made a new commit to reduce the maximum length further. 
Please guide me and review the change.

**Comment by benclifford** [comment]
ok, mostly looks good - there are a few unnecessary empty lines added that you can see here in https://github.com/Parsl/parsl/pull/3131/files in slurm.py and in db_manager.py. Those probably come from your first bit of work on this branch where you made the first changes a little bit differently than the PR that got merged (PR #3130). It would be good to remove those changes because they make this PR look more messy than it needs to be, and that can be awkward when someone looks back through git history.

File: parsl/monitoring/db_manager.py [hunk 1]
```diff
@@ -545,9 +545,11 @@ def start(self,
                     self._insert(table=STATUS, messages=reprocessable_last_resource_messages)
             except Exception:
                 logger.exception(
+
```
Comment (benclifford) [inline parsl/monitoring/db_manager.py:548]: remove this line

File: parsl/monitoring/db_manager.py [hunk 2]
```diff
@@ -545,9 +545,11 @@ def start(self,
...
                     "Exception in db loop: this might have been a malformed message, "
                     "or some other error. monitoring data may have been lost"
                 )
+
```
Comment (benclifford) [inline parsl/monitoring/db_manager.py:552]: remove this line

File: parsl/providers/slurm/slurm.py [hunk 3]
```diff
@@ -286,7 +286,9 @@ def submit(self, command: str, tasks_per_node: int, job_name="parsl.slurm") -> s
                     stdout=stdout,
                     stderr=stderr,
                     retcode=retcode
+
```
Comment (benclifford) [inline parsl/providers/slurm/slurm.py:289]: remove this line

File: parsl/providers/slurm/slurm.py [hunk 4]
```diff
@@ -286,7 +286,9 @@ def submit(self, command: str, tasks_per_node: int, job_name="parsl.slurm") -> s
...
                 )
+
```
Comment (benclifford) [inline parsl/providers/slurm/slurm.py:291]: remove this line

**Comment by Insharamin12** [comment]
Appreciate the guidance, @benclifford 
You're an amazing mentor. I've resolved the changes as per your feedback, and I hope it looks good now. 

Thank you.

**Review by benclifford — APPROVED** [review]

**Pull request merged**